src/containers/Tenant/utils/schemaQueryTemplates.ts

```diff
@@ -68,7 +68,12 @@ WITH (
 );`;
 };
 export const alterTableTemplate = (params?: SchemaQueryParams) => {
-    return `ALTER TABLE \`${params?.relativePath || '$path'}\`
+    return `-- docs: https://ydb.tech/docs/en/yql/reference/syntax/alter_table/
+
+ALTER TABLE \`${params?.relativePath || '$path'}\`
+    -- RENAME TO new_table_name
+    -- DROP COLUMN some_existing_column
+    -- ADD COLUMN text_column Utf8 NOT NULL DEFAULT 'default_value'
```

Comment:
There is an error with such query

<img width="1411" alt="Screenshot 2024-11-05 at 14 18 00" src="https://github.com/user-attachments/assets/31bb8638-ead7-451f-8583-c473b93b9a4a">

Reply:
it will be supported by default on 24-4
currently disabled
I have fixed the template